KlipTok.de-DE.json

```diff
@@ -2,43 +2,115 @@
 	// Components
 
 	// Clip
-	"Recorded": "{0} aufgenommen",
+	"Recorded": "{0} aufgenommen", // {0} ist der relative Zeitstempel der Aufnahme: "vor 3 Tagen"
 	"Likes": "Gefällt mir",
 	"Comments": "Kommentare",
 	"ClipViews": "Aufrufe",
-	"ShareThisClip": "Teile diesen Clip",
+	"ShareThisClip": "Diesen Clip teilen",
 	"KlipTokContentPlaceholder": "KlipTok Platzhalter",
 
 	// Comments
-	"ByLineFormat": "{1} hat am {0} geschrieben:",
+	"ByLineFormat": "{1} hat {0} geschrieben:", // {0} ist der relative Zeitstempel des Kommentars, {1} ist der Name des Kommentierenden
 	"Reply": "Antworten",
 
 	// Login View
 	"DarkMode": "Dunkles Design",
-	"LoginWithTwitch": "Einloggen mit Twitch",
+	"LoginWithTwitch": "Mit Twitch Einloggen",
 	"Logout": "Ausloggen",
 
 	// Header
 	"SearchForChannel": "Kanal suchen...",
-	"SearchNotFoundFormat": "{0} nicht gefunden",
+	"SearchNotFoundFormat": "Keine Ergebnisse für {0}", // {0} ist der Suchbegriff, der nicht gefunden wurde
 
 	// Sidebar
 	"SuggestedStreamers": "Vorgeschlagene Streamer",
-	"FollowedStreamers": "Streamer, denen du folgst",
+	"FollowedStreamers": "Streamer, denen ich folge",
 	"AllStreamers": "Alle Streamer",
 	"About": "Über uns",
 	"Close": "Schließen",
 
 	// Streamer Page
-	"ClickToFollow": "Klicken um zu Folgen",
-	"ClickToUnfollow": "Klicken um zu Entfolgen",
-	"ActionOnTwitchTemplate": "{0} auf Twitch",
+	"ClickToFollow": "Auf Twitch Folgen",
+	"ClickToUnfollow": "Auf Twitch Entfolgen",
+	"ActionOnTwitchTemplate": "{0} auf Twitch", // {0} wird ersetzt mit "Follow" oder "Unfollow" -> "Follow on Twitch"
 	"LiveOnTwitch": "Jetzt live auf Twitch!",
-	"ProfilePicTemplate": "{0} Profilbild",
+	"ProfilePicTemplate": "Profilbild von {0}", // {0} ist der Display Name des  Streamers
 	"TotalClipsOnKlipTok": "Clips gesamt auf KlipTok",
 	"TotalLikesAcrossAllClips": "Gefällt Mir gesamt",
 	"TotalCommentsAcrossAllClips": "Kommentare gesamt",
 	"TotalViewsAcrossAllClips": "Aufrufe gesamt",
-	"LastClipAdded": "Letzter Clip hinzugefügt:",
-	"NoProfilePicAvailable": "Kein Profilbild verfügbar"
+	"LastClipAdded": "Neuster Clip:",
+	"NoProfilePicAvailable": "Kein Profilbild verfügbar",
+
+
+	// Layout
+	"PublicBeta": "PUBLIC BETA",
+	"LoggedOutFromKlipTokAndTwitch": "Von KlipTok und Twitch ausgeloggt",
+	"LoginToKlipTokWithTwitch": "Mit Twitch bei KlipTon anmelden",
+	"AuthenticationIsNoLongerValid": "Deine Authentifizierung mit Twitch und KlipTok ist nicht mehr gültig. Bitte melde dich mit Twitch an, um fortzufahren.",
+	"Loading": "Laden...",
+
+	// Errors
+	"ErrorWhileLoading": "Fehler beim Laden.",
+	"RetryButton": "Erneut versuchen",
+
+	// Follows page
+	"TwitchChannelsIFollow": "Kanäle, denen ich folge",
+	"NotLoggedIn": "Bitte melde dich an, um Kanäle anzuzeigen, denen du folgst.",
+
+	// Home Page
+	"DiscoverPopularClips": "Beliebte Clips entdecken",
+	"DiscoverClips": "Clips entdecken",
+	"InsertCoin": "Insert Coin",
```

Comment:
Not really sure what this si supposed to mean, appears to be some sort of a loading indicator?

Reply:
Yes.. the "InsertCoin" is a loading indicator that references 1980's arcade machines